.gitmodules

```diff
@@ -0,0 +1,9 @@
+[submodule "js-codemod"]
+	path = js-codemod
+	url = https://github.com/cpojer/js-codemod.git
+[submodule "5to6-codemod"]
+	path = 5to6-codemod
+	url = https://github.com/5to6/5to6-codemod.git
+[submodule "js-import-sort"]
+	path = js-import-sort
+	url = https://github.com/Amwam/js-import-sort.git
```

Comment:
I'm not really familiar with git submodules. What is the advantage of doing it this way, rather than adding these dependencies to `package.json`?

I'm also not sure this will work correctly when published to npm, since the dependencies are not declared in `package.json`.

Reply:
@nolanlawson Hey there. Trying to add the 2nd NPM was being a pain, because the NPM for that name is 7 years old, but the code is maybe 2-3 years old. The 1st one was being a pain during troubleshooting the 1st one, so I had both repos added as submodules; we use a similar approach for work stuff (albeit not making NPMs).

Reply:
@nolanlawson I added resolutions for the submodules to package.json

Reply:
@unquietwiki For code that's on github but not npm, you can add it to the package.json as a git/github dependency. E.g. here: 

https://github.com/nolanlawson/cjs-to-es6/blob/23ea316d7970cedb45c759f3eaaeee2d6845fd10/package.json#L26

I would prefer to remove the submodules entirely. Otherwise we may end up with a situation where the code works locally but not when we publish to npm.

Reply:
@nolanlawson Hey, I got it fixed. https://www.npmjs.com/package/next-js-codemod had the newer version of that codemod as a package; the older module was missing the newer transforms.